openapi/consumed/lollipop_consumer_example.yaml

```diff
@@ -0,0 +1,216 @@
+    LollipopUserIdHeader:
+      name: x-pagopa-lollipop-user-id
+      in: header
+      description: User's fiscal code.
+      required: true
+      schema:
+        $ref: "#/components/schemas/LollipopUserId"
+    LollipopPublicKeyHeader:
+      name: x-pagopa-lollipop-public-key
+      in: header
+      description: User's public key
+      required: true
+      schema:
+        $ref: "#/components/schemas/LollipopPublicKey"
+
+    # --------------------------------------------
+    # Standard HTTP Message Signatures headers
+    # --------------------------------------------
+    LollipopContentTypeHeader:
+      name: content-type
+      in: header
+      description: The content type, if any.
+      required: false
+      schema:
+        $ref: "#/components/schemas/LollipopContentType"
+    LollipopContentDigestHeader:
+      name: content-digest
+      in: header
+      description: The body digest, if any, as defined in https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-digest-headers-10
+      required: false
+      schema:
+        $ref: "#/components/schemas/LollipopContentDigest"
+
+    LollipopSignatureInputHeader:
+      name: signature-input
+      in: header
+      description: The signature input, needed to verify the `signature` header
+      required: true
+      schema:
+        $ref: "#/components/schemas/LollipopSignatureInput"
+    LollipopSignatureHeader:
+      name: signature
+      in: header
+      description: The signature of the HTTP request, signed by the client with its private key.
+      required: true
+      schema:
+        $ref: "#/components/schemas/LollipopSignature"
+
+  schemas:
+    LollipopMethod:
+      type: string
+      enum:
+        - GET
+        - POST
+        - PUT
+        - PATCH
+        - DELETE
+    LollipopOriginalURL:
+      type: string
+      pattern: "^https:\/\/"
+    LollipopAssertionType:
+      type: string
+      enum:
+        - SAML
+        - OIDC
+    LollipopAssertionRef:
+      oneOf:
+        - $ref: "#/components/schemas/AssertionRefSha256"
+        - $ref: "#/components/schemas/AssertionRefSha384"
+        - $ref: "#/components/schemas/AssertionRefSha512"
+    LollipopUserId:
+      type: string
+      description: User's fiscal code.
+      format: FiscalCode
+      x-import: "@pagopa/ts-commons/lib/strings"
+    LollipopPublicKey:
+      type: string
+      description: represents a Base64url encode of a JWK Public Key
+      format: JwkPublicKey
+      x-import: "@pagopa/ts-commons/lib/jwk"
+    LollipopJWTAuthorization:
+      type: string
+      minLength: 1
+
+    LollipopContentType:
+      type: string
+      enum:
+        - application/json
+        - application/octet-stream
+    LollipopContentDigest:
+      type: string
+      pattern: "^(sha-256=:[A-Za-z0-9+/=]{44}:|sha-512=:[A-Za-z0-9+/=]{88}:)$"
+
+    LollipopSignatureInput:
+      type: string
+      pattern: "^(((sig[0-9]{1,})=.*?)(, ){0,1}){1,}$"
```

Comment:
am I wrong or we should use `;` instead of `,` in the `signature-input` and `signature` headers

Reply:
Also don't we need to match other fields such as `created`, `nonce` and `keyId`?